docs/configuration.rst

```diff
@@ -1902,6 +1902,7 @@ an absolute path (on the web server) or an URL.
 
 Default value: ``needs.json``
 
+You can choice option ``needs_lut.json`` as detail :ref:`needs_lut_build_json`
```

Comment:
Please check such sentences with Grammarly or other online tools for correctness.